cmd/dep/status.go

```diff
@@ -352,8 +352,8 @@ func runStatusAll(ctx *dep.Ctx, out outputter, p *dep.Project, sm gps.SourceMana
 
 		logger.Println("Checking upstream projects:")
 
-		for _, proj := range slp {
-			logger.Println(proj.Ident().ProjectRoot)
+		for i, proj := range slp {
+			logger.Printf("(%d/%d) %s\n", i+1, len(slp), proj.Ident().ProjectRoot)
```

Comment:
No, but there is not really any "computation" going on. Slice lengths are stored in the slice internal struct representation.